repair/repair.cc

```diff
@@ -279,6 +280,28 @@ repair_status tracker::get(int id) {
     }
 }
 
+future<repair_status> tracker::get_status_until_finished(int id, std::chrono::steady_clock::time_point timeout) {
```

Comment:
It's a mistake that we have to use steady_clock_type, but it's a seastar bug. We'll fix it later.